resources/assets/components/InboxItem/index.js

```diff
@@ -11,6 +11,14 @@ class InboxItem extends React.Component {
     }
   }
 
+  calculateAge(date) {
```

Comment:
@sbsmith86 I guess it's possible that it's stored in Northstar, but it is not part of the response we get. I definitely think it might be useful to be more accessible, but I'm not sure where it should live then? would it go on `app.js`?